src/libraries/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -640,6 +641,16 @@ public void ToStringTest(string name)
             Assert.Equal(assemblyName.FullName, assemblyName.ToString());
         }
 
+        [Theory]
+        [InlineData("")]
```

Comment:
This can be Fact instead since it has just one `InlineData` and the test name suggests that it is only testing EmptyName.